Build/Docker/Dockerfile

```diff
@@ -6,17 +6,13 @@ RUN echo ========== Install dependencies ========== \
     clang \
     gdb \
     git \
+    libavahi-compat-libdnssd-dev \
+    libssl-dev \
     make \
     perl \
     unzip \
     wget \
-    libasound2-dev \
-    libavahi-compat-libdnssd-dev \
-    libfaac-dev \
-    libopus-dev \
-    libsqlite3-dev \
-    libssl-dev \
-    openssh-server \
```

Comment:
Why are we removing `openssh-server` from the list. It does get automatically installed as part of other packages as dependency but good to explicitly install it.

Reply:
it's not required for building, so why have it installed? or is there another use for it that i'm missing?

Reply:
`make shell` lets you ssh into the docker container, which requires openssh to be installed.

Reply:
fair enough. never used it myself, so wasn't even aware of it.

Reply:
added back, verified that `make shell` works. PTAL.